constraints.txt

```diff
@@ -1,2 +1,3 @@
 numpy>=1.20.0
 ipython<8.13;python_version<'3.9'
+python-subunit<1.4.3
```

Comment:
There was a new version of stestr recently uploaded that fixes this incompatibility so the constraint is no longer needed. The Qiskit PR that did similar to workaround the incompatibility was simply closed as a result. The Opt PR that similar dropped that too qiskit-community/qiskit-optimization#559 though it is limiting stestr version (which strictly is not needed - you can see comments there).

Reply:
Removed